cmd/drand-cli/cli.go

```diff
@@ -125,6 +125,11 @@ var periodFlag = &cli.StringFlag{
 	Usage: "period to set when doing a setup",
 }
 
+var catchupPeriodFlag = &cli.StringFlag{
+	Name:  "catchup-period",
+	Usage: "minimum period while in catchup. set only by leader of initial setup",
```

Comment:
IF we want that in the group file, we should provide a default value to avoid explicitly requiring that

Reply:
the default when not set is 0, which is current behavior. i'll add that to usage message